app/build.gradle

```diff
@@ -116,6 +116,9 @@ dependencies {
     //mifos passcode
     implementation "com.mifos.mobile:mifos-passcode:$mifosPasscodeVersion"
 
+    //multidex
```

Comment:
@MigDinny Remove this and pull the recent changes.

Reply:
@MigDinny You have not pulled the changes properly. Check out this PR (https://github.com/openMF/mifos-mobile/pull/915/files or any other updated PR). It doesn't contain it.